icons/pretzel.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M5.3,18l5.5-5.2c1.6-1.6,1.6-4.1,0-5.7s-4.1-1.6-5.7,0C4.4,8,4,9,4,10" />
+  <path d="M18.7,18l-5.5-5.2c-1.6-1.6-1.6-4.1,0-5.7s4.1-1.6,5.7,0C19.6,8,20,9,20,10" />
+  <path d="M20,10c0,4.4-3.6,8-8,8s-8-3.6-8-8" />
```

Comment:
This can be represented by a single line:
```suggestion
  <path d="m5.3 18 5.5-5.2c1.6-1.6 1.6-4.1 0-5.7a3.98 3.98 0 0 0-5.7 0C4.4 8 4 9 4 10c0 4.4 3.6 8 8 8s8-3.6 8-8c0-1-.4-2-1.1-2.9a3.98 3.98 0 0 0-5.7 0 3.98 3.98 0 0 0 0 5.7l5.5 5.2" />
```

Reply:
```suggestion
  <path d="m5.3 18 5.5-5.2A4 4 0 1 0 4 10a8 8 0 0 0 16 0 4 4 0 1 0-6.8 2.8l5.5 5.2"/>
```